Makefile

```diff
@@ -45,32 +45,37 @@ endif
 SHLIB_LINK += -lv8
 ifdef V8_OUTDIR
 SHLIB_LINK += -L$(V8_OUTDIR)
+else
+SHLIB_LINK += -lv8_libplatform
 endif
 
+
 # v8's remote debugger is optional at the moment, since we don't know
 # how much of the v8 installation is built with debugger enabled.
 ifdef ENABLE_DEBUGGER_SUPPORT
 OPT_ENABLE_DEBUGGER_SUPPORT = -DENABLE_DEBUGGER_SUPPORT
 endif
 
-ifeq ($(OS),Windows_NT)
-	OPTFLAGS = -O2 -std=c++11 -fno-rtti
-else
-	GCCVERSION = $(shell gcc --version | grep ^gcc | sed 's/^.* //g' | cut -f1-2 -d.)
-	ifeq ($(GCCVERSION),4.6)
-		OPTFLAGS = -O2 -std=gnu++0x -fno-rtti
-	else
-		OPTFLAGS = -O2 -std=c++11 -fno-rtti
-	endif
-endif
+# for older g++ (e.g. 4.6.x), which do not support c++11
+#OPTFLAGS = -O2 -std=gnu++0x -fno-rtti
+
+OPTFLAGS = -O2 -std=c++11 -fno-rtti
```

Comment:
it was part of the original makefile for older versions of gcc - assuming it was meant to try to lower the library size.